### PR TITLE
IBX-8686: Ibexa Commerce Product Guide added in Developer Documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [IBX-8686](https://issues.ibexa.co/browse/IBX-8686)
| Versions      | master, 4.6+

Ibexa Commerce Product Guide added in Developer Documentation

Preview: https://ez-systems-developer-documentation--2460.com.readthedocs.build/en/2460/ibexa_products/ibexa_commerce/

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/ibexa_products/ibexa_commerce.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+## Capabilities and benefits
+
+[[= product_name_com =]] includes the greatest capabilities to accelerate your digital transformation: Personalized workflows, corporate account management, customized catalogs, and more.
+
+### Core features
+
+[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.
+
+#### Order management
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:44]: If we're mentioning order management - maybe we should mention Shipping and Payment management as well?

File: docs/ibexa_products/ibexa_commerce.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+#### Merchandizing
+
+Merchandizing assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
+With this feature you can engage your customers with eye-catching graphics and information
+The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.
+You can also use recommendations and campaigns to increase average order values and income.
+
+#### eCommerce
+
+With the [eCommerce feature]([[= user_doc =]]/personalization/use_cases/#ecommerce) you can transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:62]:
```suggestion
#### Product recommendations

Use [Ibexa Personzalition]([[= user_doc =]]/personalization/use_cases/#ecommerce) to transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
```
I wouldn't use the name "eCommerce feature" - here you're describing the Personalization use case for eCommerce.

File: docs/ibexa_products/ibexa_commerce.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+No matter how complicated your product or sales process are, you can present your offer in an approachable way.
+Creating engaging and personalized shopping experiences with targeted offers and recommendations, helps you to boost sales.
+Within eCommerce, product recommendations can assist users in finding the exact item that meets their needs.
+Recommendations can be used to propose related, alternative, or complimentary products to users who are unsure what to buy.
+
+![eCommerce](ecommerce.png)
+
+### Use cases
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:70]: I think all of these below (except for "eCommerce") are just features. For example: "Relevant faceted search" is not an use case, it's a feature you can use to achieve the usecase ("display products to the user").

I'd focus on two "high level" use cases:
- B2C: I want to sell my products directly to the Customers - how can Ibexa Commerce help me? With Site Factory, Page Builder, PIM, ..., faceted search, ..., connect, Personalization, (lots of options here!)
- B2B: I want to sell my products to other Companies -> how can Ibexa Commerce help me? Almost the same as above, but expanded additionaly with Customer Portals, Customer Groups - which are tailored at B2B use cases and building the long-term relations with the customers(you don't have a Sales Rep in B2C, but you do in B2B).

I really like the use cases we have in https://ez-systems-developer-documentation--2460.com.readthedocs.build/en/2460/ibexa_products/headless/#use-cases for Headless- it describes the real-life situations when someone might consider using Headless (and it even mentions when to use Ibexa Commerce - to finally become a store and sell online - so I'd expand on that).

File: docs/ibexa_products/ibexa_commerce.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+When searching for orders you can use filters, so you save time and reduce effort.
+
+![Order management](order_management.png)
+
+#### Merchandizing
+
+Merchandizing assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
+With this feature you can engage your customers with eye-catching graphics and information
+The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.
+You can also use recommendations and campaigns to increase average order values and income.
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:58]: 1) The whole section below is about recommendations, I wouldn't mix in in here as well
2) What are campaings? If I wanted to learn more about them - could we link to another part of the doc where they are described?
3) I'd mention the Storefront here - as a set of ready to use components that are included out of the box and that can be used to develop the websites faster (reducing time to market = 🏆 )

File: docs/ibexa_products/ibexa_commerce.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+title: Ibexa Commerce
+description: Discover Ibexa Commerce and its essential features, capabilities and benefits.
+month_change: true
+---
+
+# Ibexa DXP Commerce edition product guide
+
+## What is [[= product_name_com =]]
+
+[[= product_name_com =]] is one of the edition offered by [[= product_name =]].
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/ibexa_products/ibexa_commerce.md:11]: I like the latter

File: docs/ibexa_products/ibexa_commerce.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+To start using [[= product_name_com =]] you need to purchase a product license.
+For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
+You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).
+
+## How it works
+
+### Technical backstage
+
+With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/ibexa_products/ibexa_commerce.md:28]: 50/50
"that uses" is a preferred replacement of "using" (we'd rather not use gerunds)

File: docs/ibexa_products/ibexa_commerce.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+[[= product_name_com =]] is designed for complex enterprises, so it enables you to design flawless sales experiences even if your business model has built-in limitations, like distribution, price, and so on. Thanks to this, [[= product_name_com =]] is fully customizable.
+You can create online store that really fits your needs.
+
+#### Immersive shopping experiences
+
+The customer experience takes an important step further by facilitating financial transactions through the use of powerful, individualized product suggestions provided by [[= product_name_com =]] and unique pricing for various customer groups.
+
+#### Relevant faceted search
+
+Search becomes crucial, when your product catalog is extensive.
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/ibexa_products/ibexa_commerce.md:83]: Faceted search is a PIM feature.

File: docs/ibexa_products/ibexa_commerce.md (hunk 3 above)
Comment (dabrt) [reply docs/ibexa_products/ibexa_commerce.md:70]: Agree! Let's put Merchandising , consumer-facing web store and B2B the three use cases.

File: docs/ibexa_products/ibexa_commerce.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+### Technical backstage
+
+With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.
+
+[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.
+
+Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).
+
+[[= product_name_com =]] is built on [[[= product_name_headless =]]](headless.md) and includes all bundles, APIs, and [features that comes with Headless edition](headless.md#core-features) as well as [Experience features](experience.md#core-features).
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:34]: I've combined your suggestions.
[[= product_name_com =]] is built on [[[= product_name_ext =]]](experience.md) and includes all bundles, APIs, and features that come with both [Headless](headless.md#core-features) and [Experience](experience.md#core-features) editions.

File: docs/ibexa_products/ibexa_commerce.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+The Order management module allows users to review order details and updates, track order completion status, and cancel orders that are created when store customers purchase products, for a smooth purchasing process.
+You can also assign and maintain rights so that only authorized users and make modifications.
+When searching for orders you can use filters, so you save time and reduce effort.
+
+![Order management](order_management.png)
+
+#### Merchandizing
+
+Merchandizing assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
+With this feature you can engage your customers with eye-catching graphics and information
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:56]: This is not a requirement.

File: docs/ibexa_products/ibexa_commerce.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
+You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).
+
+## How it works
+
+### Technical backstage
+
+With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.
+
+[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:30]: This part was copied from another Product Guides (Headless basically) to have it consistent.
I'll change it here and there (in another PR).

File: docs/ibexa_products/ibexa_commerce.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:83]: It's presented as a use case in Merchadizing. For me makes sense.

File: docs/ibexa_products/ibexa_commerce.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+#### Order management
+
+With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can easily manage your customized purchase process.
+The Order management module allows users to review order details and updates, track order completion status, and cancel orders that are created when store customers purchase products, for a smooth purchasing process.
+You can also assign and maintain rights so that only authorized users and make modifications.
+When searching for orders you can use filters, so you save time and reduce effort.
+
+![Order management](order_management.png)
+
+#### Merchandizing
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:53]: Regarding 1. In Product Guides, I believe, we don't need to follow any page structure/order. Anyway, it's not a problem, I can move it.
Regarding 2. Core features for Commerce, following ibexa.co, are: eCommerce, Merchandizing, and Order Management.
I also see Merchandizing as a core feature, not just use case.
image

File: docs/ibexa_products/ibexa_commerce.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+# Ibexa DXP Commerce edition product guide
+
+## What is [[= product_name_com =]]
+
+[[= product_name_com =]] is one of the edition offered by [[= product_name =]].
+This edition assists you in managing each aspect of the customer journey by combining content, customization, and commerce functions into a single, dedicated interface.
+[[= product_name_com =]] offers companies a streamlined, unified platform where they can personalize each component of the online shopping experience.
+You can completely revamp your online stores with and give your consumers exceptional purchasing experiences, from first contact to post-purchase support.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:14]: I prefer to use "you" to address the user directly. I rewrote the entire section.

File: docs/ibexa_products/ibexa_commerce.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+## How it works
+
+### Technical backstage
+
+With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.
+
+[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.
+
+Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:32]: Link will be added in the PR that focuses on reorganization and unification (https://issues.ibexa.co/browse/IBX-8590).

File: docs/ibexa_products/ibexa_commerce.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:56]: My comment was regarding breaks in the preview.

[the remark this answers is not in the document]
Comment (mnocon) [reply docs/ibexa_products/ibexa_commerce.md:56]: Ok, my comment was only about the missing divide between these two sentences - either adding dot or a line break is good enough for me.

Please note that Tomek also suggests adding a comma into this sentence

File: docs/ibexa_products/ibexa_commerce.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+
+To start using [[= product_name_com =]] you need to purchase a product license.
+For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
+You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).
+
+## How it works
+
+### Technical backstage
+
+With an active lincese, you can start the [installation process](install_ibexa_dxp.md) that uses the Composer.
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:30]:
```suggestion
With an active license, you can start the [installation process](install_ibexa_dxp.md) that uses the Composer.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Payment management
+
+The [Payment](payment.md) component allows users to search for payment methods and payments, create new and manage existing payments and payment methods, and filter search results. Users can also enable or disable payment methods, change payment details, and cancel payments, depending on their role.
+
+![Payment management](payment_management.png)
+
+#### Shipping management
+
+With the [Shipping](shipping_management.md) component ussers can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods. Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:65]:
```suggestion
With the [Shipping](shipping_management.md) component users can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods. Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+[[= product_name_com =]] comes with all the features you need to launch and manage your web store, like storefront starter kit, real-time cart management, stock inventory, catalog management, and more.
+
+This edition is designed for complex enterprises and is fully customizable. It enables you to design flawless sales experiences regardless of the complexity of your business model.
+You can create online store that really fits your needs.
+
+#### Increase B2B sales
+
+[[= product_name_com =]] contains the best B2B features to help you speed your digital transformation, such as corporate account management, tailored catalogs, customized workflows, effortless reordering, custom pricing, and more.
+
+#### Automate business processess
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:118]:
```suggestion
#### Automate business processes
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 17]
```diff
@@ -0,0 +1,123 @@
...
+Products can be sorted according to a variety of criteria using faceted search.
+The value it brings makes it a crucial component in merchandizing. You can set up your search engine using [[= product_name_com =]] to help clients find what they're looking for more easily, which could result in more purchases.
+
+#### Catalog management
+
+[[= product_name_com =]] gives you the ability to manage your product repository - [PIM](pim_guide.md), and construct an infinite number of product catalogs, each with unique prices, to further customize the experience for your customers.
+
+#### Transactional emails
+
+Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through [Actito](https://www.actito.com/en-BE/) gateway to your end-users. These emails includes notifications about changes in the status of various actions taken in relation to your commerce presence.
```
Comment (mnocon) [inline docs/ibexa_products/ibexa_commerce.md:86]:
```suggestion
Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through [Actito](https://www.actito.com/en-BE/) gateway to your end-users. These emails include notifications about changes in the status of various actions taken in relation to your commerce presence.
```

**Review by mnocon — APPROVED** [review]
Thank you, looks great!

I've only marked a couple of small mistakes (mostly typos)

File: docs/ibexa_products/ibexa_commerce.md [hunk 18]
```diff
@@ -0,0 +1,123 @@
...
+
+# Ibexa DXP Commerce edition product guide
+
+## What is [[= product_name_com =]]
+
+[[= product_name_com =]] is the most powerful edition offered by [[= product_name_base =]].
+
+It assists you in managing each aspect of your customers' journey by combining content management, customization, and commerce functions into a single, dedicated solution.
+
+[[= product_name_com =]] offers a streamlined, unified platform where you can personalize each component of the online shopping experience.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:15]:
```suggestion
[[= product_name_com =]] offers a streamlined, unified platform where you can personalize each aspect of the online shopping experience.
```
Preferential, but "component" ssounds material, while aspect can also include emotions and sensations.

File: docs/ibexa_products/ibexa_commerce.md [hunk 19]
```diff
@@ -0,0 +1,123 @@
...
+You can completely revamp your online stores and give your consumers exceptional purchasing experiences, from first contact to post-purchase support.
+
+![Ibexa Commerce](ibexa_commerce.png)
+
+## Availability
+
+To start using [[= product_name_com =]] you need to purchase a product license.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:22]:
```suggestion
To start using [[= product_name_com =]], you need to purchase a product license.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 20]
```diff
@@ -0,0 +1,123 @@
...
+With [[= product_name_com =]] you can focus on accelerating your transformation into a fully-fledged eCommerce.
+It comes with all the necessary tools: customized catalogs, integration with CDP, personalized checkout workflows, payment gateway integration, transactional emails, and more.
+
+### Core features
+
+[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.
+
+#### Order management
+
+With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can you can manage orders with ease.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:50]:
```suggestion
With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can manage orders with ease.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 21]
```diff
@@ -0,0 +1,123 @@
...
+Depending on your permissions, you can search for orders, review their details and updates, track completion status, and cancel orders that are created when store customers purchase products.
+When searching for orders, you can use filters to save time and reduce effort.
+Order management is strongly connected with other components of the Commerce offering, such as [Cart](cart.md) and [Checkout](checkout.md), so users can speed up the process by uploading an order list or repeating previous transactions.
+
+![Order management](order_management.png)
+
+#### Payment management
+
+The [Payment](payment.md) component allows users to search for payment methods and payments, create new and manage existing payments and payment methods, and filter search results. Users can also enable or disable payment methods, change payment details, and cancel payments, depending on their role.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:59]:
```suggestion
The [Payment](payment.md) component allows users to search for payment methods and payments, create new and manage existing payments and payment methods, and filter search results.
Users can also enable or disable payment methods, change payment details, and cancel payments, depending on their role.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 22]
```diff
@@ -0,0 +1,123 @@
...
+
+![Payment management](payment_management.png)
+
+#### Shipping management
+
+With the [Shipping](shipping_management.md) component users can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods. Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:65]:
```suggestion
With the [Shipping](shipping_management.md) component users can create and manage shipments, search for shipments, filter search results, and define and manage various shipping methods.
Depending on their role, users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 23]
```diff
@@ -0,0 +1,123 @@
...
+
+![Shipping management](shipping_management.png)
+
+#### Storefront
+
+The [Storefront](storefront.md) package includes a starter kit for developers. It is a foundational set of components that developers can customize and extend to create their own web store implementations.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:71]:
```suggestion
The [Storefront](storefront.md) package includes a starter kit for developers.
It is a foundational set of components that developers can customize and extend to create their own web store implementations.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 24]
```diff
@@ -0,0 +1,123 @@
...
+It contains default UI components and widgets that can be modified to [create customized web store](customize_storefront_layout.md).
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:72]:
```suggestion
It contains default UI components and widgets that can be modified to [create a customized web store](customize_storefront_layout.md).
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 25]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Relevant faceted search
+
+Search becomes crucial, when your product catalog is extensive.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:76]:
```suggestion
Search becomes crucial when your product catalog is extensive.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 26]
```diff
@@ -0,0 +1,123 @@
...
+Products can be sorted according to a variety of criteria using faceted search.
+The value it brings makes it a crucial component in merchandizing. You can set up your search engine using [[= product_name_com =]] to help clients find what they're looking for more easily, which could result in more purchases.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:78]:
```suggestion
The value it brings makes it a vital component in merchandizing.
You can set up your search engine with [[= product_name_com =]] to help clients find what they're looking for more easily, which could result in more purchases.
```
Preferential, but you've used "crucial" two sentences prior.

File: docs/ibexa_products/ibexa_commerce.md [hunk 27]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Catalog management
+
+[[= product_name_com =]] gives you the ability to manage your product repository - [PIM](pim_guide.md), and construct an infinite number of product catalogs, each with unique prices, to further customize the experience for your customers.
+
+#### Transactional emails
+
+Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through [Actito](https://www.actito.com/en-BE/) gateway to your end-users. These emails include notifications about changes in the status of various actions taken in relation to your commerce presence.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:86]:
```suggestion
Commerce allows you to send transactional emails - messages that [[= product_name_base =]] can send through the [Actito](https://www.actito.com/en-BE/) gateway to your end-users.
These emails include notifications about changes in the status of various actions taken in relation to your commerce presence.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 28]
```diff
@@ -0,0 +1,123 @@
...
+With this feature you can also [create email campaigns](transactional_emails.md##create-email-campaigns) to engage users and increase sales.
+
+### Use cases
+
+#### Create personalized shipping experience
+
+Use [Ibexa Personalization]([[= user_doc =]]/personalization/use_cases/#ecommerce) to transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
+No matter how complicated your product or sales process are, you can present your offer in an approachable way.
+Creating engaging and personalized shopping experiences with targeted offers and recommendations, helps you to boost sales.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:95]:
```suggestion
Creating engaging and personalized shopping experiences with targeted offers and recommendations helps you boost sales.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 29]
```diff
@@ -0,0 +1,123 @@
...
+Within eCommerce, product recommendations can assist users in finding the exact item that meets their needs.
+Recommendations can be used to propose related, alternative, or complimentary products to users who are unsure what to buy.
+
+#### Use effective merchandising
+
+Merchandising assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
+You can engage your customers with eye-catching graphics and information.
+The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:103]:
```suggestion
The search engine makes it easy to find what they're looking for by providing quick and easy access to the product catalogs.
```
?

File: docs/ibexa_products/ibexa_commerce.md [hunk 30]
```diff
@@ -0,0 +1,123 @@
...
+
+The customer experience takes an important step further by facilitating financial transactions through the use of powerful, individualized product suggestions provided by [[= product_name_com =]] and unique pricing for various customer groups.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:105]: further -> forward ?

File: docs/ibexa_products/ibexa_commerce.md [hunk 31]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Launch consumer-facing web stores
+
+[[= product_name_com =]] comes with all the features you need to launch and manage your web store, like storefront starter kit, real-time cart management, stock inventory, catalog management, and more.
+
+This edition is designed for complex enterprises and is fully customizable. It enables you to design flawless sales experiences regardless of the complexity of your business model.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:111]:
```suggestion
This edition is designed for complex enterprises and is fully customizable.
It enables you to design flawless sales experiences regardless of the complexity of your business model.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 32]
```diff
@@ -0,0 +1,123 @@
...
+You can create online store that really fits your needs.
+
+#### Increase B2B sales
+
+[[= product_name_com =]] contains the best B2B features to help you speed your digital transformation, such as corporate account management, tailored catalogs, customized workflows, effortless reordering, custom pricing, and more.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:116]:
```suggestion
[[= product_name_com =]] contains the best B2B features to help you speed up your digital transformation, such as corporate account management, tailored catalogs, customized workflows, effortless reordering, custom pricing, and more.
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 33]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Automate business processes
+
+With [[= product_name_com =]] automation process becomes easier, what's essential if your company wants to do more with less effort.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:120]:
```suggestion
With [[= product_name_com =]], the automation process becomes easier, which is essential if your company wants to do more with less effort. 
```

File: docs/ibexa_products/ibexa_commerce.md [hunk 34]
```diff
@@ -0,0 +1,123 @@
...
+#### Increase B2B sales
+
+[[= product_name_com =]] contains the best B2B features to help you speed your digital transformation, such as corporate account management, tailored catalogs, customized workflows, effortless reordering, custom pricing, and more.
+
+#### Automate business processes
+
+With [[= product_name_com =]] automation process becomes easier, what's essential if your company wants to do more with less effort. 
+It comes with all capabilities, including order and inventory management, customer data and custom pricing, which are needed to achieve it.
+You can integrate with over 1,300 standard apps, including your CRM, ERP, PIM and DAM systems, and build custom connectors.
+Also, you can use ready-to-use pre-designed templates.
```
Comment (dabrt) [inline docs/ibexa_products/ibexa_commerce.md:123]:
```suggestion
Also, you can use the ready-to-use, pre-designed templates.
```

**Review by dabrt — APPROVED** [review]

File: docs/ibexa_products/ibexa_commerce.md (hunk 30 above)
Comment (julitafalcondusza) [reply docs/ibexa_products/ibexa_commerce.md:105]: Both correct, but 'forward' sounds better.

**Pull request merged**